packages/firebase_database/example/test_driver/firebase_database_e2e_test.dart

```diff
@@ -1,5 +1,3 @@
-// @dart=2.9
+// import 'package:e2e/e2e_driver.dart' as e2e;
 
-import 'package:e2e/e2e_driver.dart' as e2e;
-
-void main() => e2e.main();
+// void main() => e2e.main();
```

Comment:
Why is this commented?

Reply:
I thought that this package had issues currently with null-safety.